python/ql/src/semmle/python/dataflow/new/internal/DataFlowPublic.qll

```diff
@@ -372,19 +377,27 @@ newtype TContent =
   /** An object attribute. */
   TAttributeContent(string attr) { attr = any(Attribute a).getName() }
 
+/**
+ * A data-flow value can have associated content.
+ * If the value is a collection, it can have elements,
+ * if it is an object, it can have attribute values.
+ */
 class Content extends TContent {
   /** Gets a textual representation of this element. */
   string toString() { result = "Content" }
 }
 
+/** The content of an element of a list. */
```

Comment:
I might be misunderstanding, but shouldn't this be `The element of a list`? (and likewise for the other QLDocs introduced in this PR).

If it's the content of an element of a list, for the code `[foo]` I would expect this to represent the _content of `foo`_, which I think is incorrect?

Reply:
It depends on how you read "content of", I probably mean "value of", but I am happy to simply remove that possible ambiguity.

Reply:
seems like we had this discussion before, so good that we could just reuse the old way of documenting this :smile: